<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>573</width>
    <height>390</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>IP do servidor</string>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>481</width>
      <height>271</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="45,55">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QLineEdit" name="lineEdit_ip"/>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QPushButton" name="pushButton_Connect">
               <property name="text">
                <string>Connect</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_Disconnect">
               <property name="text">
                <string>Disconnect</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QListWidget" name="listWidget"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_Update">
         <property name="text">
          <string>Update</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="99,1">
           <item>
            <widget class="QSlider" name="horizontalSlider_Timing">
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="maximum">
              <number>3</number>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QPushButton" name="pushButton_Start">
             <property name="text">
              <string>Start</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_Stop">
             <property name="text">
              <string>Stop</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Plotter" name="widget" native="true"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>573</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Plotter</class>
   <extends>QWidget</extends>
   <header location="global">plotter.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_Connect</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>connecting()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>76</x>
     <y>100</y>
    </hint>
    <hint type="destinationlabel">
     <x>21</x>
     <y>91</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_Disconnect</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>disconnecting()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>207</x>
     <y>95</y>
    </hint>
    <hint type="destinationlabel">
     <x>242</x>
     <y>96</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_Update</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>updating()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>88</x>
     <y>246</y>
    </hint>
    <hint type="destinationlabel">
     <x>237</x>
     <y>231</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_Timing</sender>
   <signal>valueChanged(int)</signal>
   <receiver>label</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>147</x>
     <y>283</y>
    </hint>
    <hint type="destinationlabel">
     <x>194</x>
     <y>284</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_Start</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>starting()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>82</x>
     <y>317</y>
    </hint>
    <hint type="destinationlabel">
     <x>92</x>
     <y>341</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_Stop</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>stopping()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>217</x>
     <y>311</y>
    </hint>
    <hint type="destinationlabel">
     <x>216</x>
     <y>341</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>connecting()</slot>
  <slot>disconnecting()</slot>
  <slot>updating()</slot>
  <slot>starting()</slot>
  <slot>stopping()</slot>
 </slots>
</ui>
